ale_linters/go/golangci_lint.vim

```diff
@@ -12,8 +12,8 @@ function! ale_linters#go#golangci_lint#GetCommand(buffer) abort
 
     if l:lint_package
         return ale#path#BufferCdString(a:buffer)
-        \   . '%e run '
-        \   .  l:options
+        \   . '%e run *.go'
```

Comment:
Hey, may I ask you to provide me an example where this kind of file selection is needed? I never had issues without it in the past and the the docs of golangci indicate that a simple `run` should be enough: https://github.com/golangci/golangci-lint#quick-start

Reply:
Hey, I just have trouble with `golangci-lint` when I'm working on my project, and I found this option in the docs. But seems `ale` did not support this option properly.

Reply:
Can you please point me to the docs where you saw this?

Reply:
```
g:ale_go_golangci_lint_package                 g:ale_go_golangci_lint_package
                                               b:ale_go_golangci_lint_package
  Type: Number
  Default: 0

  When set to 1, the whole Go package will be checked instead of only the
  current file.
```

Reply:
Ah sorry then I got you wrong. Yeah I use this option set to 1 on a daily basis and never had any issues. But you’re using macOS right? Which project you’re using does not seem to work for you?

Reply:
If you compare the code, you will find this option made nothing different.